src/test/java/pl/piekoszek/gorskimatches/facebook/FacebookControllerTest.java

```diff
@@ -0,0 +1,205 @@
+package pl.piekoszek.gorskimatches.facebook;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import pl.piekoszek.gorskimatches.equation.EquationRandomizer;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class FacebookControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private FacebookMessageService facebookMessageService;
+
+    @MockBean
+    private EquationRandomizer equationRandomizer;
+
+    @Value("${VERIFY_TOKEN}")
+    private String VERIFY_TOKEN;
+
+    @Test
+    public void shouldReturnForbiddenStatusWhenTokenIsWrong() throws Exception {
+        mockMvc.perform(get("/api/webhook/facebook/page/message")
+                        .param("hub.verify_token", "a")
+                        .param("hub.challenge", "a")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .accept(MediaType.APPLICATION_JSON))
+                .andExpect(status().isForbidden());
+    }
+
+    @Test
+    public void shouldReturnOkWhenTokenIsCorrect() throws Exception {
+        mockMvc.perform(get("/api/webhook/facebook/page/message")
+                        .param("hub.verify_token", VERIFY_TOKEN)
+                        .param("hub.challenge", "a")
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .accept(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk()).andReturn().getResponse().getContentAsString();
```

Comment:
`.andReturn().getResponse().getContentAsString();` and do what with this `String`?